week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item-sum.component.js

```diff
@@ -1,7 +1,7 @@
-import { div } from '../framework/dom-creators.js'
+import { div } from '../framework/dom-creators.js';
 
 export function cartItemSum({ value, currency = 'PLN' }) {
```

Comment:
Brakuje Ci odpowiednich zapisów w `tsconfig.json` tak aby ustalić katalog `dist` do którego projekt będzie się budował. Tak to pliki `.js` i pliki `.ts` mieszają się w kodzie i na repo.

Plików `.js` generowanych przez TypeScript nie powinno być na repo.
Build powinien być w `.gitignore` - żeby to osiągnąc potrzebujesz budowania do `/dist` na przykład.

sprawdź:
https://www.typescriptlang.org/tsconfig#outDir